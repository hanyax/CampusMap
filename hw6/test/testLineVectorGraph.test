# This test tests the Graph structor of a line where node were
# all linked with unique edge and formed a line vector

CreateGraph test

AddEdge test n1 n2 e1
AddEdge test n2 n1 e1
AddEdge test n2 n3 e2
AddEdge test n3 n2 e2
AddEdge test n3 n4 e3
AddEdge test n4 n3 e3

FindPath test n1 n4

FindPath test n4 n1
