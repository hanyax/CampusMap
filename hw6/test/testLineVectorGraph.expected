# This test tests the Graph structor of a line where node were
# all linked with unique edge and formed a line vector

created graph test

added edge e1 from n1 to n2 in test
added edge e1 from n2 to n1 in test
added edge e2 from n2 to n3 in test
added edge e2 from n3 to n2 in test
added edge e3 from n3 to n4 in test
added edge e3 from n4 to n3 in test

path from n1 to n4:
n1 to n2 via e1
n2 to n3 via e2
n3 to n4 via e3

path from n4 to n1:
n4 to n3 via e3
n3 to n2 via e2
n2 to n1 via e1
